### DISCUSSION TITLE
Using link for card header

### DISCUSSION THREAD

**Discussion by shyamajp** [twilio-labs/paste#1266] [Q&A] [answered]
Hello!
I am using <Heading> inside <Card> and <Heading> is surrounded by <Link>(from react-router-dom).

What I want to achieve here is to make <Heading> in $text-color-link since it has link to other pages. Small note: There are multiple cards in this page, each with link to a different page.

I tried color props, didn't work here. So the only option is overwrite color in css as in the snippet below? Is there a way to import all the css variables at once to the project? (such as $text-color-link)

I appreciate any suggestions! Thank you.

jsx
```
<Card padding='space60'>
  <Link to={uri}>
    <Heading as='h4' variant='heading40'>
      This is title
    </Heading>
  </Link>
  <Paragraph>This is description</Paragraph>
</Card>
```

scss
```
$text-color-link: rgb(2, 99, 224);

article {
  a {
    text-decoration-color: $text-color-link;
    h4 {
      color: $text-color-link;
    }
  }
}
```

**Comment by richbachman** [comment, marked as answer]
Hi @shyamajp 

Thanks or posting your question. Our components will block any extra styles that are passed. This helps maintain consistency and encapsulated styles. So a Heading will always look like any other Heading.

You could use our Anchor component with the Heading component like so:

```
<Heading as="h4" variant="heading40">
    <Anchor href={url}>
        This is the title
    </Anchor>
</Heading>
```

That way you get the Anchor styles. Our Anchor should be able to work with  React Router. Take a look a this previous discussion for a couple helpful links / options: https://github.com/twilio-labs/paste/discussions/758#discussioncomment-78990

Let us know if that helps.

**Reply by shyamajp** [reply]
Thank you for your response, this is very helpful.
Just want to clarify but there is no way to import all css variables like colors, correct?

**Reply by shyamajp** [reply]
I would also like to hear any suggestions regarding this header with link for a better UI, ensuring that users know that this is a link.

**Reply by richbachman** [reply]
> Just want to clarify but there is no way to import all css variables like colors, correct?

Correct. The only way to "style" things within Paste is to use the [Box](https://paste.twilio.design/primitives/box) and [Text](https://paste.twilio.design/primitives/text) primitives which come with style props ties to the [design tokens](https://paste.twilio.design/tokens).

> I would also like to hear any suggestions regarding this header with link for a better UI, ensuring that users know that this is a link.

Maybe add an Anchor at the bottom of the card? Something like this: https://share.getcloudapp.com/OAugeqoy

**Reply by shyamajp** [reply]
Sorry for the delay, I will change the design to anchor at the bottom as recommended by Paste team. Thank you for all your support :)

**Comment by jweintraub** [comment]
Can you share the design for this modal?
